pkg/psmdb/statefulset.go

```diff
@@ -631,3 +710,32 @@ func PodTopologySpreadConstraints(cr *api.PerconaServerMongoDB, tscs []corev1.To
 	}
 	return result
 }
+
+type caRef struct{ Name, Key string }
```

Comment:
Do we really need a new type for this? Can't we use `SecretKeySelector`, since it contains the same fields?